examples/Cliner/generation.json

```diff
@@ -0,0 +1 @@
{\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2636, \"end\": 2650}, \"_tid\": 40, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2694, \"end\": 2704}, \"_tid\": 41, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2733, \"end\": 2741}, \"_tid\": 42, \"cliner_type\": \"problem\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2749, \"end\": 2755}, \"_tid\": 43, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2801, \"end\": 2808}, \"_tid\": 44, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2830, \"end\": 2836}, \"_tid\": 45, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2858, \"end\": 2865}, \"_tid\": 46, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2886, \"end\": 2905}, \"_tid\": 47, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2927, \"end\": 2945}, \"_tid\": 48, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 2968, \"end\": 2984}, \"_tid\": 49, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 3002, \"end\": 3019}, \"_tid\": 50, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 3048, \"end\": 3057}, \"_tid\": 51, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 3081, \"end\": 3088}, \"_tid\": 52, \"cliner_type\": \"treatment\"}}, {\"py/object\": \"ft.onto.clinical.ClinicalEntityMention\", \"py/state\": {\"_span\": {\"py/object\": \"forte.data.span.Span\", \"begin\": 3126, \"end\": 3137}, \"_tid\": 53, \"cliner_type\": \"treatment\"}}], \"generics\": [], \"replace_back_operations\": [], \"processed_original_spans\": [], \"orig_text_len\": 3337, \"serialization\": {\"next_id\": 54}}}"
```

Comment:
You have serialized the file wrong. A correctly serialized JSON file will not have all the backslashes.

Reply:
It is updated and the backslashes have gone.